staging/src/k8s.io/client-go/util/workqueue/queue.go

```diff
@@ -209,7 +205,7 @@ type Typed[t comparable] struct {
 	shuttingDown bool
 	drain        bool
 
-	metrics queueMetrics
+	metrics queueMetrics[t]
```

Comment:
just for reference, this lowercase `t` is the generic  type`comparable` , not the defined `type` `interface{}`